### PR TITLE
DCS-1871: 🚀 Enable additional prisons in production

### PR DESCRIPTION
**Pull request by connormaglynn**

### CODE REVIEW

File: helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml [hunk 1]
```diff
@@ -30,6 +30,7 @@ generic-service:
     REDIS_TLS_ENABLED: "true"
     TOKEN_VERIFICATION_ENABLED: "true"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY);IngestionEndpoint=https://northeurope-0.in.applicationinsights.azure.com/;LiveEndpoint=https://northeurope.livediagnostics.monitor.azure.com/"
+    ENABLED_PRISONS: 'BFI,BNI,CDI,DMI,EXI,EYI,HEI,HLI,LCI,LEI,LII,LPI,LWI,MHI,NMI,NWI,PNI,PVI,WCI,WSI,WWI'
```
Comment (connormaglynn) [inline helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml:33]: Didn't see much benefit in maintaining three separate lists now we're rolling out to more prisons. So I've moved this to the generic values file and enabled prisons are consistent across the environments.

File: helm_deploy/README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ version.BuildInfo{Version:"v3.0.1", GitCommit:"7c22ef9ce89e0ebeb7125ba2ebf7d421f
 
 - Ensure a TLS cert for your intended hostname is configured and ready, see section below.
 
-### Useful helm (v3) commands:
```
Comment (connormaglynn) [inline helm_deploy/README.md:14]: Was a [NBSP](https://en.wikipedia.org/wiki/Non-breaking_space), which meant the title wasn't displaying as expected

Comment (connormaglynn) [reply helm_deploy/README.md:14]: Think this might have come from the template. I'll check that as well

**Review by andrewrlee — APPROVED** [review]
lgtm

**Pull request merged**